fedimint-core/src/config.rs

```diff
@@ -190,6 +191,26 @@ impl ClientConfig {
     pub fn federation_id(&self) -> FederationId {
         self.global.federation_id()
     }
+
+    /// Get the value of a given meta field
+    pub fn meta<V: serde::de::DeserializeOwned>(&self, key: &str) -> anyhow::Result<Option<V>> {
+        let Some(str_value) = self.global.meta.get(key) else {
+            return Ok(None);
+        };
+        serde_json::from_str(str_value).context(format!("Decoding meta field '{key}' failed"))
+    }
+
+    /// Creates an API client for the federation
+    pub fn api(&self) -> DynGlobalApi {
+        DynGlobalApi::from_config(self)
+    }
+
+    pub fn invite_code(&self, peer: &PeerId) -> Option<InviteCode> {
```

Comment:
Documentation needed: what does the `peer` argument do?

Reply:
Done